zerver/lib/remote_server.py

```diff
@@ -325,6 +325,10 @@ def send_analytics_update_realm_properties(
         # this endpoint; don't rashly mark push notifications as
         # disabled when they are likely working perfectly fine.
         return
+    except JsonableError as e:
+        analytics_logger = logging.getLogger("zulip.analytics")
+        analytics_logger.warning(e.msg)
+        return
```

Comment:
What's the background for this change?

Reply:
`send_analytics_to_push_bouncer` previously handled error like `JsonableError(_("Duplicate registration detected."))`

Reply:
Oh I see, test_push_notifications,py tests. Hmm.